lighthouse-cli/chrome-launcher.ts

```diff
@@ -31,17 +31,17 @@ const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
 const isWindows = process.platform === 'win32';
 
-class ChromeLauncher {
+export class ChromeLauncher {
```

Comment:
just learning here, what difference does this make to TS as opposed to `export {ChromeLauncher}`